posts/2015-07-25-new-javascript-generators.md

```diff
@@ -0,0 +1,120 @@
+---
+title: New JavaScript Generators
+author: Freezeboy
+date: 2015-07-25 00:00
+toc: true
+---
+
+# Summary
+
+With the upcoming Servant version, it will be way easier to build a javascript
+connector to your webservice. The chosen API is *as usual with Servant*
+putting the magic away from your eyes to let you concentrate on what you are
```

Comment:
Again, I'd avoid "magic" if that was my post. That's just my opinion, you don't necessarily *have to* take it into account.